lib/components/Shell/Shell.tsx

```diff
@@ -64,16 +64,24 @@ export function Shell({ theme, masthead, navigation, children, onClick }: ShellP
 
     return (
         <ThemeProvider theme={shellTheme}>
-            <div className={css('shell')} onClick={onClick}>
-                {masthead && <Masthead navigation={navigation} {...masthead} />}
-                <div className={css('nav-and-workspace')}>
-                    {navigation && <Navigation {...navigation} />}
-                    <section role='main' className={css('workspace')}>
-                        {children}
-                    </section>
-                    <ContextPanelRoot />
+            <>
+                <div className={css('shell')} onClick={onClick}>
+                    {masthead && <Masthead navigation={navigation} {...masthead} />}
+                    <div className={css('nav-and-workspace')}>
+                        {navigation && <Navigation {...navigation} />}
+                        <section role='main' className={css('workspace')}>
+                            {children}
+                        </section>
+                        <ContextPanelRoot />
+                    </div>
                 </div>
-            </div>
+                <div 

```

Comment:
can we use the portal instead?

Reply:
using the portal makes it so that there are N 'dropdown-containers' in our DOM, even if only one is being used at a time, apart from that for live regions to work the live region should always be present in the DOM and the content of the live region is the one that should change.

We could rewrite this component to actually take advantage of portals and use them properly, but the current approach is not accessible.